smtpd/spfwalk.c

```diff
@@ -129,6 +130,17 @@ dispatch_record(struct asr_result *ar, void *arg)
 	}
 }
 
+int
+validate_address(int af, const char *s)
+{
+	struct in6_addr ina;
+
+	if (strrchr(s, '/') != NULL)
```

Comment:
Do we need to check explicitly check for prefix notation? How about... https://nimmagadda.net/walk.diff

Reply:
Good catch on the wrong size argument to inet_net_pton, thanks. Your patch looks much cleaner than mine, and rereading the inet_net_pton(3) man page, I think it should work (though I haven't tested it). You may want to consider applying an analogous change to the function `host_v4` in `src/pfctl/pfctl_parser.c` (lines 1747--1753) off of which my patch was based.